PixelHunter1995/PixelHunter1995.csproj

```diff
@@ -35,7 +38,7 @@
     <DebugSymbols>true</DebugSymbols>
     <DebugType>full</DebugType>
     <Optimize>false</Optimize>
-    <OutputPath>bin\$(MonoGamePlatform)\$(Platform)\$(Configuration)\</OutputPath>
+    <OutputPath>bin\Windows\x86\Debug\</OutputPath>
```

Comment:
Why did this change? And why to x86? Mostly just curious.

Reply:
Good question, I don't know. I used the graphical tool (project -> pixelhunter properties) to add a platform and make it 64-bit.
I can however guess - I think it happened because I touched the output path, as it lost the "Windows" part, so I quickly re-added it. This must have somehow made it lose the variable path, and become an explicitly set one like that instead.